ERCS/erc-725.md

```diff
@@ -37,270 +37,249 @@ And the event:
+#### `setDataBatch`
 
 ```solidity
-function executeBatch(uint256[] memory operationsType, address[] memory targets, uint256[] memory values, bytes[] memory datas) external payable returns(bytes[] memory)
+function setDataBatch(bytes32[] memory dataKeys, bytes[] memory dataValues) external payable
 ```
 
-Function Selector: `0x31858452`
+Function Selector: `0x97902421`
 
-Executes a batch of calls on any other smart contracts, transfers the blockchain native token, or deploys a new smart contract.
+Sets array of data at multiple data keys. MUST only be called by the current owner of the contract.
 
 _Parameters:_
 
-- `operationsType`: the list of operations type used to execute.
-- `targets`: the list of addresses to call. `targets` will be unused if a contract is created (operation types 1 and 2).
-- `values`: the list of native token amounts to transfer (in Wei).
-- `datas`: the list of call data, or the creation bytecode of the contract to deploy.
+- `dataKeys`: the data keys which values to set.
+- `dataValues`: the array of bytes to set.
 
 _Requirements:_
 
-- Parameters array MUST have the same length.
+- Array parameters MUST have the same length.
 - MUST only be called by the current owner of the contract.
-- MUST revert when the execution or the contract creation fails.
-- `target` SHOULD be address(0) in case of contract creation with `CREATE` and `CREATE2` (operation types 1 and 2).
-- `value` SHOULD be zero in case of `STATICCALL` or `DELEGATECALL` (operation types 3 and 4).
-
-_Returns:_ `bytes[]` , array list of returned data of the called function, or the address(es) of the contract deployed (operation types 1 and 2).
 
-**Triggers Event:** [ContractCreated](#contractcreated), [Executed](#executed) on each call iteration
+**Triggers Event:** [DataChanged](#datachanged)
 
-### `ERC725X` Events
+### Events
 
-#### Executed
+#### `DataChanged`
 
 ```solidity
-event Executed(uint256 indexed operationType, address indexed target, uint256 indexed value, bytes4 data);
+event DataChanged(bytes32 indexed dataKey, bytes dataValue)
 ```
 
-MUST be triggered when `execute` creates a new call using the `operationType` `0`, `3`, `4`.
-
-#### ContractCreated
+MUST be triggered when a data key is successfully set.
 
-```solidity
-event ContractCreated(uint256 indexed operationType, address indexed contractAddress, uint256 indexed value, bytes32 salt);
-```
+### Y Data keys
 
-MUST be triggered when `execute` creates a new contract using the `operationType` `1`, `2`.
+Data keys, are the way to retrieve values via `getData()`. These `bytes32` values can be freely chosen, or defined by a standard.
+A common way to define data keys is the hash of a word, e.g. `keccak256('ERCXXXMyNewKeyType')` which results in: `0x6935a24ea384927f250ee0b954ed498cd9203fc5d2bf95c735e52e6ca675e047`
 
----
+The LSP2 ERC-725 JSON Schema standard is a more explicit ERC-725 Y data key standard, that defines key types and value types, and their encoding and decoding.
```

Comment:
> LSP2 ERC-725 JSON Schema

This is an external link (I think?). You should either remove it or replicate the contents here (either inline or in your assets directory).